lib/screens/home/home_screen.dart

```diff
@@ -34,6 +34,10 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
   }
 
   Future<void> _initData() async {
```

Comment:
I don't think we need Future and async here 🤔

Reply:
Thank you, I updated all at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/71/commits/0f3a00be29e7718111d39be827ed1ec2da631dbd 🙏